packages/core/stories/segmented-button-group/segmented-button-group.stories.tsx

```diff
@@ -60,3 +72,33 @@ export const Icons: StoryFn<SegmentedButtonGroupProps> = () => {
     </StackLayout>
   );
 };
+
+export const WithMenu: StoryFn<SegmentedButtonGroupProps> = () => {
```

Comment:
It might be quicker to create a short Split Button pattern example rather than place it here